be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/WitnessValidator.scala

```diff
@@ -17,25 +16,19 @@ object WitnessValidator {
 }
 
 sealed class WitnessValidator(dbActorRef: => AskableActorRef) extends MessageDataContentValidator {
+  def bindToPipe[T](rpcMessage: JsonRpcMessage, opt: Option[T], pipelineError: PipelineError): GraphMessage = {
+    if (opt.isEmpty) Left(pipelineError) else Right(rpcMessage)
+  }
```

Comment:
This will can be moved to an outer class (probably `MessageDataContentValidator`) in a later PR, but I didn't want to make that one heavier than it already is